Dockerfile

```
@@ -45,30 +45,13 @@ RUN apk add --no-cache ruby \
   "asciidoctor-pdf:${ASCIIDOCTOR_PDF_VERSION}"
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-# Haskell build for: erd
+# Install erd-go (https://github.com/kaishuu0123/erd-go) as replacement for erd (https://github.com/BurntSushi/erd)
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-# Alpine 3.16 is the last known working version for building erd (dynamic linking)
-FROM alpine:3.16 AS erd-builder
+FROM golang:alpine as erd-builder
```

```suggestion
FROM golang:1.12-alpine as erd-builder
```

Rationale : https://github.com/kaishuu0123/erd-go/blob/v2.0.0/Dockerfile#L1 specifies golang 1.12.
Better using the same version as the 2.0.0 of erd-go was used, avoiding bad surprises when golang is updated.